src/app/pages/about/about.component.html

```diff
@@ -184,6 +184,36 @@ <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
     </div>
     <hr/>
 
+    <!-- Anthony's Row-->
+    <div class="row d-flex">
+
+      <!-- Anthony's Image -->
+      <div class="col-md-2 order-md-2 image-container">
+        <img alt="Picture of Anthony Zhou" src="{{ Anthony.regularLink }}" class="img-fluid regular-picture">
+        <img alt="Silly picture of Anthony Zhou" src="{{ Anthony.sillyLink }}" class="img-fluid silly-picture">
+      </div>
+
+      <!-- Anthony's Blurb -->
+      <div class="col-md-7 order-md-3 mt-3 mt-md-0">
+        <p class="text-justify">Anthony is the daring bug squasher and legal eagle at Virtrolio. While known for his
+          spectacular puns and passion for extremely long hikes, he also enjoys building things ~with love~! Anthony is
+          currently studying Software Engineering at the University of Waterloo. In his spare time, he likes to play the
+          piano and travel (RIP). He also writes about math things, lands planes, and has made it out of REV alive.</p>
+      </div>
+
+      <!-- Anthony's Contact Info. -->
+      <div class="col-md-3 order-md-1">
+        <p><strong>Contact Anthony:</strong></p>
+        <ul>
+          <li>LinkedIn: <a href="https://www.linkedin.com/in/anthonyftwang" target="_blank">Anthony Wang</a></li>
+          <li>Github: <a href="https://www.github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
```

Comment:
GitHub doesn't use www
```suggestion
          <li>Github: <a href="https://github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
```